server/src/test/java/org/elasticsearch/common/logging/JsonThrowablePatternConverterTests.java

```diff
@@ -74,18 +81,10 @@ private String format(LogEvent event) {
         converter.format(event, builder);
         String jsonStacktraceElement = builder.toString();
 
-        return """
-            {
-              "type": "console",
-              "timestamp": "2019-01-03T16:30:53,058+0100",
-              "level": "DEBUG",
-              "component": "o.e.a.s.TransportSearchAction",
-              "cluster.name": "clustername",
-              "node.name": "node-0",
-              "cluster.uuid": "OG5MkvOrR9azuClJhWvy6Q",
-              "node.id": "VTShUqmcQG6SzeKY5nn7qA",
-              "message": "msg msg"
-              %s
-            }""".formatted(jsonStacktraceElement);
+        return "{\"type\": \"console\", \"timestamp\": \"2019-01-03T16:30:53,058+0100\", \"level\": \"DEBUG\", "
+            + "\"component\": \"o.e.a.s.TransportSearchAction\", \"cluster.name\": \"clustername\", \"node.name\": \"node-0\", "
+            + "\"cluster.uuid\": \"OG5MkvOrR9azuClJhWvy6Q\", \"node.id\": \"VTShUqmcQG6SzeKY5nn7qA\",  \"message\": \"msg msg\" "
+            + jsonStacktraceElement
+            + "}";
```

Comment:
We could maybe try something a bit like this if we still wanted the text block?

```suggestion
        return """
            {
              "type": "console",
              "timestamp": "2019-01-03T16:30:53,058+0100",
              "level": "DEBUG",
              "component": "o.e.a.s.TransportSearchAction",
              "cluster.name": "clustername",
              "node.name": "node-0",
              "cluster.uuid": "OG5MkvOrR9azuClJhWvy6Q",
              "node.id": "VTShUqmcQG6SzeKY5nn7qA",
              "message": "msg msg"
              %s
            }""".formatted(jsonStacktraceElement).lines().map(String::trim).collect(Collectors.join(" "));
```

Reply:
I am not sure. in a way it hides the intend that it is mean to be one line

Reply:
Good point. You could do:

```suggestion
        return """
            {"type": "console", "timestamp": "2019-01-03T16:30:53,058+0100", "level": "DEBUG", \
            "component": "o.e.a.s.TransportSearchAction", "cluster.name": "clustername", "node.name": "node-0", \
            "cluster.uuid": "OG5MkvOrR9azuClJhWvy6Q", "node.id": "VTShUqmcQG6SzeKY5nn7qA",  "message": "msg msg" \
            %s }""".formatted(jsonStacktraceElement);
```

Reply:
I agree with @pgomulka on this one. The original text is definitely a one-liner.